zap/src/main/java/org/zaproxy/zap/extension/httppanel/view/syntaxhighlight/HttpPanelSyntaxHighlightTextArea.java

```diff
@@ -129,6 +132,18 @@ public HttpPanelSyntaxHighlightTextArea() {
         this.setFont(
                 FontUtils.getFontWithFallback(FontType.workPanels, this.getFont().getFontName()));
 
+        if (DisplayUtils.isDarkLookAndFeel()) {
+            try {
+                Theme theme =
+                        Theme.load(
+                                getClass()
+                                        .getResourceAsStream(
+                                                "/org/fife/ui/rsyntaxtextarea/themes/dark.xml"));
+                theme.apply(this);
+            } catch (IOException e) {
+                log.error("Failed to set RSyntaxTextArea dark theme", e);
```

Comment:
Warn ? ZAP is still going to function fine right? (Just might not look the greatest)

Edit maybe also move the theme application outside the try block? (Or add a conditional) no point trying to apply if there was an exception reading the file...

Reply:
I'm easy on warn vs error.
But I cant see the point of trying to load the theme unless we think we need to :)

Reply:
Agreed, so keep it inside `if (DisplayUtils.isDarkLookAndFeel())` but outside the try, so that there isn't an attempt to apply something that couldn't be read?